behaviour/graql/language/undefine.feature

```diff
@@ -270,3 +281,24 @@ Feature: Graql Undefine Query
       undefine person sub entity;
       """
     Then the integrity is validated
+
+
+  Scenario: undefine an entity type throws if instances exist
+
+  Scenario: undefine a relation type throws if instances exist
+
+  Scenario: undefine an attribute type throws if instances exist
+
+  Scenario: a type can be undefined after deleting all of its instances
+
+  Scenario: undefine 'plays' from type throws if there is an instance playing that role in an existing relation
+
+  Scenario: 'relates' can be undefined when there are existing relations, but none of them have that roleplayer
+
+  Scenario: undefine 'relates' from relation type throws if there is an existing instance with that roleplayer
+
+  Scenario: 'has' can be undefined when there are existing instances, but none of them have that attribute
```

Comment:
i generally like to use the term "ownership" or "owns" for having an attribute, for future reference, if you think that is also a good term to use!

Reply:
Yeah I'm also seeing it used extensively in the ITs since starting to migrate them

Reply:
Updated to
```
Scenario: attribute ownership can be undefined when there are instances of the owner, but none of them own that attribute

Scenario: undefine attribute ownership throws if any instance of the owner currently owns that attribute```